src/win_delay_load_hook.c

```diff
@@ -21,10 +21,16 @@ static FARPROC WINAPI load_exe_hook(unsigned int event, DelayLoadInfo* info) {
     return NULL;
 
   if (_stricmp(info->szDll, "iojs.exe") != 0 &&
-      _stricmp(info->szDll, "node.exe") != 0)
+      _stricmp(info->szDll, "node.exe") != 0 &&
+      _stricmp(info->szDll, "node.dll") != 0)
     return NULL;
 
-  m = GetModuleHandle(NULL);
+  m = GetModuleHandle("node.dll");
```

Comment:
I updated the PR based on feedback from this thread, does this change your opinon?

https://github.com/justinmchase/node-gyp/commit/a7fcf441f4ca34f7b7a4c90f5085eca0dbaec7f3